src/components/media/responsive.js

```diff
@@ -0,0 +1,55 @@
+const media = document.querySelector("#media-content")
```

Comment:
in this file i suggest to use `function` instead of `const = () =>` and `var` for variables.

If you switch page and come back to a convo the call ui is not responsive anymore